Sources/Kingfisher+Rx.swift

```diff
@@ -14,15 +14,15 @@ extension Reactive where Base == KingfisherWrapper<ImageView> {
     public func setImage(with resource: Resource?,
                          placeholder: Placeholder? = nil,
                          options: KingfisherOptionsInfo? = nil) -> Single<Image> {
-        return Single<Image>.create { [base] single in
+        return Single.create { [base] single in
```

Comment:
We could go further and completely remove the type declaration
```suggestion
        return .create { [base] single in
```

Reply:
I agree to skip it completely if infer type 👍
We appreciate good feedback and reflect your feedback.